ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -58,6 +58,7 @@ module.exports = Marionette.LayoutView.extend({
     this.listenTo(this.model, 'resetToDefaults change:type', this.reshow)
     this.listenTo(this.model, 'change:filterTree', this.reshow)
     this.listenTo(this.model, 'closeDropdown', this.closeDropdown)
+    this.listenTo(wreqr.vent, 'resetSearch', this.cancel)
```

Comment:
❓ So this will be listening for 'resetSearch' on the `wreqr.vent` object. Is that what we want to happen? Typically, we listen to changes/updates/additions to an object we 'own', i.e. `this.model`.

Reply:
I think the vent route is best. The thing sending out the resetSearch is a React container so I'm not sure how I would go about listening to it as I would a Marionette model. The listenTo is also used throughout other parts of the UI code here so its probably best to keep to convention unless there is an issue I'm not seeing.

Reply:
Listening for event on `vent` just semantically seems pretty odd to me as it's a messaging framework, but assuming we're sunsetting this code either way I suppose it's fine for now.